Hakawai/Core/HKWTextView.m

```diff
@@ -598,6 +615,17 @@ - (void)scrollViewDidScroll:(UIScrollView *)scrollView {
     }
 }
 
+#pragma mark - UITextInput
+
+- (void)insertDictationResult:(NSArray<UIDictationPhrase *> *)dictationResult {
+    NSMutableString *string = [[NSMutableString alloc] init];
+    for (NSUInteger i = 0; i < dictationResult.count; i++) {
```

Comment:
Use fast enumeration `for (UIDictationPhrase * dictationPhrase in dictationResult)`